cg/server/admin.py

```diff
@@ -323,7 +324,15 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
```

Comment:
I would like to keep from_sample, this is used for downsampled samples and it's good to easily check that it's set for these samples.